nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
@Duhemm @asoltysik 
The issue with this macros is that it depends on the machine it was built on. We would need to publish multiple Scala Native artifacts. IMO it would be better if we could delegate this information gathering to an existing library if it exists.